### DISCUSSION TITLE
Can Revision Tool be used prior to installing ReviOS?

### DISCUSSION THREAD

**Discussion by e-t-l** [meetrevision/revision-tool#39] [Q&A]
Does the Tool depend on interacting with some aspect of the ReviOS installation process in order to function—like the AME Wizard, for example—or can it enable/disable features independently?

In other words, could this tool be deployed in order to selectively disable/reenable features on different OS flavors, like AtlasOS or even vanilla Windows?

**Comment by melo936** [comment]
> Does the Tool depend on interacting with some aspect of the ReviOS installation process in order to function—like the AME Wizard, for example—or can it enable/disable features independently?

ReviOS Playbook relies on Revision Tool in order to disable Defender and enable Superfetch for HDDs, because it isn't a good idea to maintain the same script on both repositories.

> In other words, could this tool be deployed in order to selectively disable/reenable features on different OS flavors, like AtlasOS or even vanilla Windows?

No, in the description it says that `The program is meant to be only used on ReviOS 22.10 or up`. While it is possible to use it on other systems, it is not guaranteed to work perfectly, because we have different methods of modifying some system functionality, such as Defender. Atlas or any other systems might have different ways of modifying things, so it will cause nothing but the system damage, thus it is neither recommended nor supported.
